docs/reference/indices/aliases.asciidoc

```diff
@@ -113,6 +113,10 @@ unless overriden in the request using the `expand_wildcards` parameter,
 similar to <<index-hidden,hidden indices>>. This property must be set to the
 same value on all indices that share an alias. Defaults to `false`.
 
+`must_exist`::
```

Comment:
I was not sure about the naming here. Initially I went with "required", but I felt that that was a bit too generic, so opted for a word where it's at least clear what is meant.